counter-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: ticktock
```

Comment:
It takes a PR and dog food eating to force discussion, so I'd slightly disagree with your last remark :)

Now, trying to understand your comment:
> We may want to default to applicationName.channelName at SCS level
you mean in the `Binder` code itself, right ?
>  and then skip this property entirely in the modules themselves
_i.e._ remove this entirely from this module `application.yml`? I tend to come to that same conclusion, I don't see what this serves (apart maybe from preventing IT tests from failing, as providing an `Environment` in those is not easy)